packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -168,6 +168,15 @@ export class Button extends FormAssociatedButton {
     private handleFormReset = () => {
         this.form?.reset();
     };
+
+    public content: HTMLSlotElement;
+    public contentContainer: HTMLSpanElement;
+    public handleContentChange(): void {
+        this.contentContainer.classList.toggle(
+            "content",
+            this.content.assignedNodes().length > 0
+        );
+    }
```

Comment:
I missed this before, but we want to do this in the component implementation. This can be a noop in foundation. I think we do something similar for slider label regarding orientation.